src/client/datascience/kernel-launcher/kernelProcess.ts

```diff
@@ -395,72 +385,41 @@ export class KernelProcess implements IKernelProcess {
     private async launchAsObservable(workingDirectory: string, @ignoreLogging() cancelToken: CancellationToken) {
         let exeObs: ObservableExecutionResult<string> | undefined;
 
-        // Use a daemon only if the python extension is available. It requires the active interpreter
-        if (
```

Comment:
This whole change is just removing this if condition. This ripples up because we don't use a daemon in launching anymore.